### DISCUSSION TITLE
Nuxt 3 Tailwind content option in config doesnt detect utility classes in source files

### DISCUSSION THREAD

**Discussion by ThimoDEV** [tailwindlabs/tailwindcss#10538] [Help]
![image](https://user-images.githubusercontent.com/68739517/217662902-219eb4c4-84b2-4905-b6ce-2d6699ece600.png)

I always get this warning running nuxt v3.1.2 with the tailwind nuxt config and the @nuxt/tailwind plugin. 

I tried to remove lock file, delete node modules, delete nuxt folder but that doesnt seem to solve the issue

**Comment by wongjn** [comment]
As the warning suggests, consider checking that you have used some Tailwind classes, and that the `content` file globs cover any files that use Tailwind classes in them. You could also try integrating Tailwind [via the framework guide](https://tailwindcss.com/docs/guides/nuxtjs#3).

**Reply by ThimoDEV** [reply]
I just tested it by creating a clean nuxt 3 project. With tailwind setup through the framework guide the same warning shows up. One other existing project of mine doesnt get the warning.

**Reply by wongjn** [reply]
Consider providing a remotely-hosted git repository that reproduces the unexpected behavior and we can then take a closer look.

**Reply by ThimoDEV** [reply]
https://github.com/ThimoDEV/nuxt3-tailwind-warning-content

This should be the most basic version that produces the warning

**Reply by ThimoDEV** [reply]
https://github.com/ThimoDEV/nuxt3-tailwind-warning-content/tree/tailwind-own-nuxt-warning

Here I have an example with through the framework guide install, the previous example uses @nuxt/tailwindcss

**Reply by wongjn** [reply]
As mentioned previously and as the warning suggests, no Tailwind classes have been used. Add a Tailwind class to a file covered by the `content` file globs and the warning should go away.

**Reply by ThimoDEV** [reply]
Thank you, that did the trick indeed. It does feel a bit weird if i'm honest. Why would I need to get warned if I only added the config for tailwind.?
